plugins/casutg/src/main/java/com/blazemeter/jmeter/threads/concurrency/ConcurrencyThreadGroup.java

```diff
@@ -1,32 +1,51 @@
 package com.blazemeter.jmeter.threads.concurrency;
 
-import com.blazemeter.jmeter.threads.AbstractDynamicThreadGroup;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
 import org.apache.jmeter.engine.StandardJMeterEngine;
 import org.apache.jmeter.threads.JMeterThread;
 import org.apache.jmeter.threads.ListenerNotifier;
+import org.apache.jmeter.util.JMeterUtils;
 import org.apache.jorphan.collections.ListedHashTree;
 import org.apache.jorphan.logging.LoggingManager;
 import org.apache.log.Logger;
 
+import com.blazemeter.jmeter.threads.AbstractDynamicThreadGroup;
+
 public class ConcurrencyThreadGroup extends AbstractDynamicThreadGroup {
     private static final Logger log = LoggingManager.getLoggerForClass();
-    public static final int MIN_CHECK_TIME = 1000;
+    private static final long DEFAULT_TEMPORISATION = JMeterUtils.getPropDefault("dynamic_tg.temporisation", 10L);
```

Comment:
This property will need a note in documentation, so users and us will know it exists for the future.

Reply:
I didn't find documentation for the other one "dynamic_tg.shift_rampup_start", in which file should it be documented ?

Reply:
It's this file: https://github.com/undera/jmeter-plugins/blob/master/site/dat/wiki/ConcurrencyThreadGroup.wiki
I'll add there other relevant properties after merging this PR.